docs/conf.py

```diff
@@ -118,6 +123,54 @@
 # modindex_common_prefix = []
 
 
+# Function to generate source links pointing to GitHub
+def linkcode_resolve(domain, info):
```

Comment:
Is this function magically picked up by linkcode?

Reply:
`sphinx.ext.linkcode` looks for a function called `linkcode_resolve` in the sphinx config (`conf-py`) and uses it as a callback to figure out where the source hyperlink should point to, yes.

You can try it out in the PR docs, e.g. for [imread](https://imageio--834.org.readthedocs.build/en/834/_autosummary/imageio.v3.imread.html#imageio.v3.imread) and click on the `[Source]` button next to the name of the function.

Reply:
Nice! 🚀